lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -717,6 +718,133 @@ UsdMayaJobExportArgs UsdMayaJobExportArgs::CreateFromDictionary(
     return UsdMayaJobExportArgs(allUserArgs, dagPaths, timeSamples);
 }
 
+/* static */
+MStatus UsdMayaJobExportArgs::GetDictionaryFromEncodedOptions(
```

Comment:
This whole function was moved from exportTranslator.cpp so that the logic can be shared among multiple callers.

It parses the "value1=x;value2=y" format and creates a VtDictionary from it and optionally the pair of start/end time samples.